rekolekcje-engine/src/main/resources/db/changelog/004-personal-data.xml

```diff
@@ -1,44 +1,52 @@
 <?xml version="1.1" encoding="UTF-8" standalone="no"?>
-<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
jak wyżej, nie poprawiaj formatowania "przy okazji". Na przyszłość jeśli jest źle sformatowane albo powstanie wielki bałagan, to można założyć issue, żeby posprzątać liquibase. Wtedy wszyscy będą wiedzieli, że to issue rozwali bazę, a nie commit, który ma dodać jedno pole spowoduje, że nagle wszyscy muszą się resetować.